src/app/space/create/deployments/apps/collapsible-deployment-info.component.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Component,
+  Input,
+} from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { Environment } from '../models/environment';
+
+@Component({
+  selector: 'collapsible-deployment-info',
+  templateUrl: 'collapsible-deployment-info.component.html'
+})
+export class CollapsibleDeploymentInfoComponent {
+
+  @Input() mini: boolean;
+  @Input() applicationId: string;
+  @Input() collapsed: boolean;
+  @Input() config: any;
+  @Input() chartData: any;
```

Comment:
I think some of these inputs should really just be properties "owned" by this component rather than passed to it by its parent - namely, "mini", "config", and "chartData".

Also I have a rename suggestion: collapsible-deployment-info -> deployment-details ?